volontulo/tests/views/test_offers.py

```diff
@@ -475,3 +475,165 @@ def test_for_correct_slug(self):
         self.assertIn('offer', response.context)
         self.assertIn('volunteers', response.context)
         self.assertEqual(len(response.context['volunteers']), 5)
+
+
+class TestOffersJoin(TestCase):
+    u"""Class responsible for testing offer's join page."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
+        organization = Organization.objects.create(
+            name=u'',
+            address=u'',
+            description=u'',
+        )
+        organization.save()
+
+        cls.offer = Offer.objects.create(
+            organization=organization,
+            description=u'',
+            requirements=u'',
+            time_commitment=u'',
+            benefits=u'',
+            location=u'',
+            title=u'volontulo offer',
+            time_period=u'',
+            status='NEW',
+        )
+        cls.offer.save()
+
+        volunteer = User.objects.create_user(
+            u'volunteer@example.com',
+            u'volunteer@example.com',
+            u'vol123',
+        )
+        volunteer.save()
+        cls.volunteer_profile = UserProfile(user=volunteer)
```

Comment:
As we talk yesterday, there is no need to use cls. - class attributes. Just use staticmethod decorator.